e2e/namespace/install/cli/run_test.go

```diff
@@ -129,5 +129,14 @@ func TestKamelCLIRun(t *testing.T) {
 			// Clean up
 			Expect(Kamel("delete", "--all", "-n", ns).Execute()).To(Succeed())
 		})
+		t.Run("Run with http dependency", func(t *testing.T) {
```

Comment:
Elegant way to reuse the test we already have but with a different scope. Kudos!